tests/Unit/ApplicationTest.php

```diff
@@ -4,7 +4,7 @@
 use CodeClimate\PhpTestReporter\Application;
 use Symfony\Component\Console\Tester\ApplicationTester;
 
-class ApplicationTest extends \PHPUnit_Framework_TestCase
+class ApplicationTest extends \PHPUnit\Framework\TestCase
```

Comment:
Ha, nice, wasn't aware it's been backported to `4.8`, see https://github.com/sebastianbergmann/phpunit/blob/master/ChangeLog-4.8.md#4835---2017-02-06!